languages/messages/wikia/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "CoreMessagesWikia"
+format = "mediawiki"
+source.file = "MessagesEn.php"
```

Comment:
:(

So the manual upload to Crowdin is the only solution here? A fake extension with i18n file with these messages only sounds way to hacky to me ;)